src/server/rdb_save.cc

```diff
@@ -274,14 +274,21 @@ io::Result<uint8_t> RdbSerializer::SaveEntry(const PrimeKey& pk, const PrimeValu
                                              uint64_t expire_ms, DbIndex dbid) {
   DVLOG(3) << "Selecting " << dbid << " previous: " << last_entry_db_index_;
   SelectDb(dbid);
-  uint8_t buf[16];
-  error_code ec;
+
   /* Save the expire time */
   if (expire_ms > 0) {
-    buf[0] = RDB_OPCODE_EXPIRETIME_MS;
+    uint8_t buf[16] = {RDB_OPCODE_EXPIRETIME_MS};
     absl::little_endian::Store64(buf + 1, expire_ms);
-    ec = WriteRaw(Bytes{buf, 9});
-    if (ec)
+    if (auto ec = WriteRaw(Bytes{buf, 9}); ec)
+      return make_unexpected(ec);
+  }
+
+  /* Save the key poperties */
+  uint32_t df_mask_flags = pk.IsSticky() ? DF_MASK_FLAG_STICKY : 0;
```

Comment:
why do you write RDB_OPCODE_DF_MASK and then write the mask value?
instead you could add opcode RDB_OPCODE_STICKY and write this opcode incase IsSticky() in rdb save
and in rdb load if you read this opcode set the item as sticky. Why do you need the mask?

Reply:
@romange I understand that you asked for it in the issue #251 , but what are the uses cases? what else do you think we can encode in this flag?

Reply:
my motives are to be as less intrusive as possible with opcodes we share with Redis format. 
There is no downside to have a mask, so why not have something extensible? 
(I do not have a strong opinion about this)

Reply:
just because I dont see when we will use this
anyway we can stay with this implementation